css/style.css

```diff
@@ -0,0 +1,717 @@
+@import url('https://fonts.googleapis.com/css2?family=Heebo&family=PT+Sans&family=Roboto:wght@500&family=Rowdies:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Heebo&family=PT+Sans&display=swap');
+body{
+    margin:0 auto;
+    font-family: 'Heebo', sans-serif;
+    font-family: 'PT Sans', sans-serif;
+    font-family: 'Roboto', sans-serif;
+    font-family: 'Rowdies', cursive;
+
+}
+
+.headermain{
+    height: 4rem;
+    display: flex;
+    flex-direction: row;
+    box-shadow: 0 8px 6px -6px lightblue;background-color: lightblue;
```

Comment:
```suggestion
    box-shadow: 0 8px 6px -6px lightblue;
    background-color: lightblue;
```

Reply:
Code is now indented, thanks for the comments